src/core/utils/urlutils.js

```diff
@@ -14,21 +14,20 @@ export function getLiveApiUrl (env = PRODUCTION) {
 
 /**
  * Returns the base url for the analytics backend in the desired environment.
- * @param {string} env The desired environment.
- * @param {boolean} conversionTrackingEnabled If conversion tracking has been opted into.
+ * @param {string} env The desired environment.]
+ * @param {string} cloudChoice The choice of cloud provider.
  */
-export function getAnalyticsUrl (env = PRODUCTION, conversionTrackingEnabled = false) {
-  if (isEu()) {
-    return 'https://www.eu.yextevents.com';
-  }
-  if (conversionTrackingEnabled) {
-    return env === SANDBOX
-      ? 'https://sandbox-realtimeanalytics.yext.com'
-      : 'https://realtimeanalytics.yext.com';
+export function getAnalyticsUrl (env = PRODUCTION, cloudChoice = GLOBAL_MULTI) {
+  const cloudRegionString = isEu() ? 'eu' : 'us';
+  let envAndCloudChoice = '';
+  if (env === SANDBOX && cloudChoice === GLOBAL_GCP) {
+    envAndCloudChoice = 'sandbox-gcp.';
+  } else if (env === SANDBOX) {
+    envAndCloudChoice = 'sandbox.';
+  } else if (cloudChoice === GLOBAL_GCP) {
+    envAndCloudChoice = 'gcp.';
   }
-  return env === SANDBOX
-    ? 'https://sandbox-answers.yext-pixel.com'
```

Comment:
Do we know if the yext events URLs are also the same as yext pixel?

Reply:
The doc says the following:

> This new domain will replace the functionality for 
> www.yext-pixel.com
> realtimeanalytics.yext.com
> answers.yext-pixel.com
> 
> 
> This will not replace
> pixel.yext-pub.com
> Pl.yext.com
> *.yext-wrap.com
> *.yext-wraps.com

I can confirm in the discussion thread that this is the case